staging/src/k8s.io/apiserver/pkg/apis/bulk/OWNERS

```diff
@@ -0,0 +1,17 @@
+reviewers:
```

Comment:
Please pare this list down-- many of these people won't know anything about this API. (it's easy enough to add them back!)